cache-url/lib/AmpCurlUrlGenerator.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "hex_", "argsIgnorePattern": "str" }]*/
```

Comment:
Do we need to ignore `hex_`?

Reply:
Yes, `hex_` is used by SHA256, therefore, since we are commenting it, it is considered as unused 😢